src/rules/pyupgrade/rules/unpack_list_comprehension.rs

```diff
@@ -18,7 +18,11 @@ fn contains_await(expr: &Expr) -> bool {
         ExprKind::IfExp { test, body, orelse } => {
             contains_await(test) || contains_await(body) || contains_await(orelse)
         }
-        ExprKind::Dict { keys, values } => keys.iter().chain(values.iter()).any(contains_await),
+        ExprKind::Dict { keys, values } => keys
+            .iter()
+            .flatten()
+            .chain(values.iter())
+            .any(contains_await),
```

Comment:
Oh, we should rewrite this to use `any_over_expr`. Can be a separate PR, or included here...